notebooks/tutorial.raster_zonal_stats.ipynb

```diff
@@ -114,7 +114,7 @@
   },
```

Comment:
<p>You could change the heading to <strong>Comparison of </strong><code><strong>exactextract</strong></code><strong> and </strong><code><strong>rasterstats</strong></code><strong> </strong>in general since you talk about the recommendations and percent difference later on.<strong>  </strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>